src/components/sbb-form-field/sbb-form-field.tsx

```diff
@@ -0,0 +1,192 @@
+import { Component, Element, h, Prop, State } from '@stencil/core';
+import { InterfaceSbbFormFieldAttributes } from './sbb-form-field.custom';
+import { AgnosticMutationObserver as MutationObserver } from '../../global/helpers/mutation-observer';
+
+let nextId = 0;
+
+/**
+ * @slot label - Slot to render a label.
+ * @slot prefix - Slot to render an icon on the left side of the input.
+ * @slot input - Slot to render an input.
+ * @slot suffix - Slot to render an icon on ther right side of the input.
+ * @slot error - Slot to render an error.
+ */
+@Component({
+  shadow: true,
+  styleUrl: './sbb-form-field.scss',
+  tag: 'sbb-form-field',
+})
+export class SbbFormField {
+  /**
+   * Add a specific space if the `<sbb-error>` is present.
+   */
+  @Prop() public errorSpace?: InterfaceSbbFormFieldAttributes['errorSpace'] = 'default';
+
+  /**
+   * Add a `<label>` for the input.
+   */
+  @Prop() public label: string;
+
+  /**
+   * Indicates whether the input is optional.
+   */
+  @Prop() public optional?: boolean;
+
+  /**
+   * Size variant, either l or m.
+   */
+  @Prop() public size?: InterfaceSbbFormFieldAttributes['size'] = 'l';
```

Comment:
```suggestion
  @Prop() public size?: InterfaceSbbFormFieldAttributes['size'] = 'l';
```
is not the m variant the standard?